### DISCUSSION TITLE
Random picks for adaptive polygons?

### DISCUSSION THREAD

**Discussion by gerroon** [nortikin/sverchok#4663] [Q&A] [answered]
Hi

Is it possible to randomize the donor objects for the adaptive polygons so that it is not the same object all over?

**Comment by Durman** [comment, marked as answer]
[untitled.zip](https://github.com/nortikin/sverchok/files/9571851/untitled.zip)

![image](https://user-images.githubusercontent.com/28003269/190315973-99062175-2514-4df7-a942-dee642194258.png)

**Reply by gerroon** [reply]
Thanks Durman, this might work for me. Thanks for the file.

Is there a way to retain the UVs on the created objects?

**Reply by Durman** [reply]
It's a bit twisted

[untitled.zip](https://github.com/nortikin/sverchok/files/9580837/untitled.zip)

![image](https://user-images.githubusercontent.com/28003269/190560684-b6566aa6-a2b2-43e9-a356-352ea911a5e3.png)
